lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -192,6 +212,10 @@ function dosomething_northstar_transform_user($user, $password = null) {
 
   // Set values in ns_user if they are set.
   foreach ($optional as $ns_key => $drupal_key) {
+    if (! isset($user->$drupal_key)) {
```

Comment:
Right, you can inline the `$user->{$drupal_key}` part in the `isset` so it gets caught alongside the other potentially nonexistent values:

```php
      if (isset($user->{$drupal_key}[LANGUAGE_NONE][0]['value'])) {
        $northstar_user[$ns_key] = $user->{$drupal_key}[LANGUAGE_NONE][0]['value'];
     }
   }
```